Submodule external/hello_imgui/hello_imgui contains modified content
Submodule external/imgui contains modified content
diff --git a/external/hello_imgui/hello_imgui/external/imgui/misc/freetype/imgui_freetype.cpp b/external/hello_imgui/hello_imgui/external/imgui/misc/freetype/imgui_freetype.cpp
index 68e38ed9..c2135493 100644
--- a/external/hello_imgui/hello_imgui/external/imgui/misc/freetype/imgui_freetype.cpp
+++ b/external/hello_imgui/hello_imgui/external/imgui/misc/freetype/imgui_freetype.cpp
@@ -48,7 +48,7 @@
 #ifdef IMGUI_ENABLE_FREETYPE_LUNASVG
 #include FT_OTSVG_H             // <freetype/otsvg.h>
 #include FT_BBOX_H              // <freetype/ftbbox.h>
-#include <lunasvg.h>
+#include <lunasvg/lunasvg.h>
 #if !((FREETYPE_MAJOR >= 2) && (FREETYPE_MINOR >= 12))
 #error IMGUI_ENABLE_FREETYPE_LUNASVG requires FreeType version >= 2.12
 #endif
diff --git a/external/hello_imgui/hello_imgui/hello_imgui_cmake/hello_imgui_build_lib.cmake b/external/hello_imgui/hello_imgui/hello_imgui_cmake/hello_imgui_build_lib.cmake
index 7d1e768..2a0eb22 100644
--- a/external/hello_imgui/hello_imgui/hello_imgui_cmake/hello_imgui_build_lib.cmake
+++ b/external/hello_imgui/hello_imgui/hello_imgui_cmake/hello_imgui_build_lib.cmake
@@ -444,35 +444,12 @@ function(_him_add_freetype_to_imgui)
     endif()
 
     #
-    # 2. Build lunasvg (static)
+    # 2. Build lunasvg
     #
     # Fetch and build lunasvg
-    if(NOT TARGET lunasvg)
-        # Try using lunasvg unofficial package from vcpkg
-        find_package(unofficial-lunasvg CONFIG QUIET)
-        if(unofficial-lunasvg_FOUND)
-            target_link_libraries(imgui PRIVATE unofficial::lunasvg::lunasvg)
-        elseif(NOT HELLOIMGUI_FETCH_FORBIDDEN)
-            set(backup_shared_lib ${BUILD_SHARED_LIBS})
-            set(BUILD_SHARED_LIBS OFF CACHE BOOL "" FORCE)
-            include(FetchContent)
-            FetchContent_Declare(lunasvg
-                GIT_REPOSITORY https://github.com/sammycage/lunasvg
-                GIT_TAG        v2.3.9
-                GIT_PROGRESS TRUE
-            )
-            FetchContent_MakeAvailable(lunasvg)
-            set(BUILD_SHARED_LIBS ${backup_shared_lib} CACHE BOOL "" FORCE)
-            target_link_libraries(imgui PUBLIC lunasvg)
-            get_target_property(lunasvg_include_dirs lunasvg INTERFACE_INCLUDE_DIRECTORIES)
-            # Patch lunasvg include dir, for installable version (CMake install shenanigans)
-            set_target_properties(lunasvg PROPERTIES INTERFACE_INCLUDE_DIRECTORIES $<BUILD_INTERFACE:${CMAKE_BINARY_DIR}/_deps/lunasvg-src/include>)
-            get_target_property(lunasvg_include_dirs lunasvg INTERFACE_INCLUDE_DIRECTORIES)
-
-            him_add_installable_dependency(lunasvg)
-            hello_imgui_msvc_target_set_folder(lunasvg ${HELLOIMGUI_SOLUTIONFOLDER}/external)
-        endif()
-    endif()
+    # Try using lunasvg unofficial package from vcpkg
+    find_library(LUNASVG_LIBRARY lunasvg REQUIRED)
+    target_link_libraries(imgui PRIVATE LUNASVG_LIBRARY)
 
     #
     # 3. Add freetype and LunaSvg support to imgui
Submodule external/imgui/imgui contains modified content
diff --git a/external/imgui/imgui/misc/freetype/imgui_freetype.cpp b/external/imgui/imgui/misc/freetype/imgui_freetype.cpp
index 68e38ed9..c2135493 100644
--- a/external/imgui/imgui/misc/freetype/imgui_freetype.cpp
+++ b/external/imgui/imgui/misc/freetype/imgui_freetype.cpp
@@ -48,7 +48,7 @@
 #ifdef IMGUI_ENABLE_FREETYPE_LUNASVG
 #include FT_OTSVG_H             // <freetype/otsvg.h>
 #include FT_BBOX_H              // <freetype/ftbbox.h>
-#include <lunasvg.h>
+#include <lunasvg/lunasvg.h>
 #if !((FREETYPE_MAJOR >= 2) && (FREETYPE_MINOR >= 12))
 #error IMGUI_ENABLE_FREETYPE_LUNASVG requires FreeType version >= 2.12
 #endif

